app/controllers/items_controller.rb

```diff
@@ -0,0 +1,28 @@
+require 'item_marketplace/presenters/items_index_presenter'
+require 'item_marketplace/presenters/new_item_presenter'
+
+class ItemsController < ApplicationController
+  def index
+    items = RepositoryContainer.get_repo(:item).all
```

Comment:
What happens here when your app becomes super popular and you have 10 million items for sale?